src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -207,3 +214,64 @@ async def post(self):
         self.set_status(201)
         self.set_header("Content-Type", "application/json; charset=UTF-8")
         self.write(response)
+
+    @authenticated(permissions=[Permissions.SYSTEM_UPDATE])
```

Comment:
Not that it really matters, but this should be `GARDEN_UPDATE`

Reply:
We don't have permissions setup for the Gardens. There is also another PR out to remove all of these and reduce it down to the CRUD + Admin permissions.

Reply:
That's fair. Can we just remove this from the PR then?

Reply:
I am going to leave it, just in case we don't get the other PR merged in.